traj/scripts/demo

```diff
@@ -66,16 +73,17 @@ max_jerks = np.deg2rad(np.array([
     8000.0,
     8000.0,
     16000.0,
-]))
-
-joint_names = ['joint_1', 'joint_2', 'joint_3', 'joint_4', 'joint_5', 'joint_6']
+])) * 1.0
 
 rospy.init_node('traj_demo')
 follow_trajectory_client = actionlib.SimpleActionClient('joint_trajectory_action', FollowJointTrajectoryAction)
 follow_trajectory_client.wait_for_server()
 rospy.loginfo('Connected to joint trajectory server')
 
 plot_trajectory = rospy.get_param('~plot', False)
+joint_prefix = rospy.get_param('~joint_prefix', '')
+
+joint_names = [joint_prefix + name for name in 'joint_1', 'joint_2', 'joint_3', 'joint_4', 'joint_5', 'joint_6']
 
 joint_states_sub = rospy.Subscriber('joint_states', JointState, joint_state_callback)
 while not rospy.is_shutdown() and initial_joint_states is None:
```

Comment:
Would `rospy.wait_for_message('joint_states', sensor_msgs.msg.JointState)` not work?

Reply:
Whoa - i don't think i've ever used wait_for_message() before. That's great! I'll update the code to use it.